keras2onnx/ke2onnx/dense.py

```diff
@@ -10,15 +10,7 @@
 from ..common.onnx_ops import apply_sigmoid, apply_softmax, apply_identity, apply_relu, apply_add
 from ..common.onnx_ops import apply_elu, apply_selu, apply_tanh, apply_hard_sigmoid
 import numpy as np
-
-_activation_map = {_get_activation('sigmoid'): apply_sigmoid,
-                   _get_activation('softmax'): apply_softmax,
-                   _get_activation('linear'): apply_identity,
-                   _get_activation('relu'): apply_relu,
-                   _get_activation('elu'): apply_elu,
-                   _get_activation('selu'): apply_selu,
-                   _get_activation('tanh'): apply_tanh,
-                   _get_activation('hard_sigmoid'): apply_hard_sigmoid}
+from .activation import _activation_map
```

Comment:
If it was imported by module, it should be started with an underscore.

Reply:
updated